plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -785,7 +785,7 @@ SecRule REQUEST_FILENAME "@rx /ocs/v[0-9]\.php/cloud/users(?:/[^/]+)?$" \
     ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.value"
 
 # Fix Nextcloud Cookie FPs
-SecAction \
+SecRule REQUEST_FILENAME "@beginsWith /" \
```

Comment:
Use `unconditionalMatch` instead (https://github.com/SpiderLabs/ModSecurity/wiki/Reference-Manual-%28v2.x%29#unconditionalmatch)

Reply:
@theseion Why is that better?

Reply:
Already found it [here](https://github.com/coreruleset/wordpress-rule-exclusions-plugin/pull/26).